vmdb/spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "spec_helper"
+
+describe ProviderForemanController do
```

Comment:
@AparnaKarve controller spec tests look good to me, i think it's a good start to have some tests for a explorer controller.